### ISSUE TITLE
Training old epoch with modified dataset 

### ISSUE THREAD

**Issue by CorbinFerrie** [NVlabs/Deep_Object_Pose#252]
Hi,
I've realized recently that my dataset for my training set did not include enough lowlight and closeup shots of my object. Is it possible to retrain my model using a somewhat working epoch? Like including more images in my dataset to account for the parts of my model that aren't working? 
My model is also not really accurate so I had to change some of the sensitivity parameters in the config in order to get any pose information from it. In a bit of a time crunch so I don't have time to retrain for another 5 days like last time.
Below are my attached images of what it looks like atm.
Thanks,
![BeliefMapTest2](https://user-images.githubusercontent.com/49255301/167726641-79432ea3-7a81-4c6a-9a62-6dda2f0689a5.png)
![CloseUp](https://user-images.githubusercontent.com/49255301/167726644-ae08d41e-a68a-4a2b-8b47-eaf293137973.png)
![FarAway](https://user-images.githubusercontent.com/49255301/167726647-1b3259b5-011f-4e91-a9ea-46d5ef55532b.png)

**Comment by TontonTremblay** [comment]
If you create more data, you resume the training with the more data by passing `--net` and your set of weight. There will be a moment where the loss will be high but it will kick down back.
